examples/site_selection.jl

```diff
@@ -7,10 +7,11 @@ using DataFrames
 here = @__DIR__
 dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
 
-criteria_df = ADRIA.sample(ex_domain, 5) # get scenario dataframe
+criteria_df = ADRIA.sample(dom, 5) # get scenario dataframe
 
 area_to_seed = 1.5 * 10^-6  # area of seeded corals in km^2
 ts = 5  # time step to perform site selection at
 
-sum_cover = 0.1 .* ones(5, size(dom.site_data, 1))  # initial coral cover matching number of criteria samples
-ranks = run_site_selection(dom, criteria_df[criteria_df.guided.>0, :], sum_cover, area_to_seed, ts)
+# initial coral cover matching number of criteria samples (size = (no. criteria scens, nsites))
```

Comment:
`(size = (no. criteria scens, nsites))`

Expand "nsites" to match term used for "criteria scens" (i.e., "no. of sites")